tests/preprocessing/test_clustering.py

```diff
@@ -1,9 +1,10 @@
 from tests.common import *
 
 
-def test_cluster(adata, adata_neighbors):
+def test_cluster(adata):
+    scIB.pp.reduce_data(adata, pca=True, n_top_genes=200, neighbors=True, umap=False)
     _, _, score_all, clustering = scIB.cl.opt_louvain(
-        adata_neighbors(adata),
+        adata,
```

Comment:
why not just require `adata_neighbors` here? This seems redundant with the `adata_neighbors` fixture.

Reply:
forgot about that one as well...